### PR TITLE
Target v1.1.7 release notes.

### PR DESCRIPTION
**Pull request by shtomar-adb**

### CODE REVIEW

**Comment by rymorale** [comment]
adding reviewers as @shtomar-adb needs access.

**Review by calebk1 — APPROVED** [review]
LGTM!

File: release-notes/README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## June 15, 2021
+
+### Android Target 1.1.7
+
+* Added the changes to move away from bintray and start using Sonatype to push SDK to Maven Central.
```
Comment (calebk1) [inline release-notes/README.md:11]: I'd say "... to push the SDK to Maven Central."

File: using-mobile-extensions/adobe-target/adobe-target-release-notes.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Release Notes
 
+## June 15, 2021
+
+### Android Target 1.1.7
+
+* Added the changes to move away from bintray and start using Sonatype to push SDK to Maven Central.
```
Comment (calebk1) [inline using-mobile-extensions/adobe-target/adobe-target-release-notes.md:7]: I'd say "... to push the SDK to Maven Central."

**Review by calebk1 — APPROVED** [review]
Looks good, just make the fixes below and we're fine.

**Pull request merged**